src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs

```diff
@@ -97,7 +99,7 @@ public async Task HandleAsync_GivenNoOrganisationAccess_ShouldThrowSecurityExcep
             handler = new GetAatfNotesRequestHandler(authorization, noteDataAccess, mapper, systemDataDataAccess);
 
             //act
-            var result = await Record.ExceptionAsync(() => handler.HandleAsync(GetAatfNotesRequest()));
+            var result = await Record.ExceptionAsync(() => handler.HandleAsync(GetAatfNotesRequest(SystemTime.UtcNow.Year)));
```

Comment:
create a complianceYear readonly only property and assign it in the class constructor to fixture.Create<int>(). Using SystemTime means that we are not sure if we are using the request value or SystemTime. Same change throughout the file.